### PR TITLE
UI updating of post load and bidding screens

### PR DESCRIPTION
**Pull request by Anik-Chakraborty**
Update UI of post load screen for desktop and load confirmation screens for all sizes.
Change UI of bidding Screen.

### CODE REVIEW

File: lib/Web/screens/PublishMethodBidWebScreen.dart [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+        for (int j = 0; j < transporterList.length; j++) {
+          if (areListsEqual(
+              providerData.loadTransporterList[i], transporterList[j])) {
+            selectedTransporterList.add(transporterList[j]);
+          }
+        }
+      }
+      setSelectedTransporterList = false;
+    }
+    print("ok -------> $selectedTransporterList");
```
Comment (chirag4601) [inline lib/Web/screens/PublishMethodBidWebScreen.dart:89]: remove such  comments as these won't help other developers rather confuse them

File: lib/Web/screens/PublishMethodBidWebScreen.dart [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+        padding: EdgeInsets.all(20),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            (widget.publishMethod == 'Bid')
+                ? Row(
+                    children: [
+                      BiddingDateTime(),
+                      SizedBox(
```
Comment (chirag4601) [inline lib/Web/screens/PublishMethodBidWebScreen.dart:149]: Like this BiddingDateTime, can we replace the static code which follows this line

File: lib/Widgets/MyLoadsCard.dart [hunk 3]
```diff
@@ -89,29 +90,141 @@ class MyLoadsCard extends StatelessWidget {
...
+                              loadDetailsScreenModel.unloadingPointCity ?? 'Null',
+                              textAlign: TextAlign.center,
+                              style: TextStyle(
+                                  color: kLiveasyColor,
+                                  fontSize: textFontSize,
+                                  fontFamily: 'Montserrat'),
+                            )
+                          ],
+                        ),
+                        (loadDetailsScreenModel.unloadingPointCity2 != 'NA')?
```
Comment (chirag4601) [inline lib/Widgets/MyLoadsCard.dart:195]: stacking code like this in one line is never helpful and is not a good practice.
Use the command cltr+alt+L to format the code

File: lib/Widgets/addTransporterWidget.dart [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+      headers: <String, String>{
+        'Content-Type': 'application/json; charset=UTF-8',
+      },
+    );
+    if (response.statusCode == HttpStatus.ok) {
+      print("Transporter added successfully");
+      Navigator.of(diaLogContext).pop();
+    }
+  } catch (e) {
+    print('1 $e');
```
Comment (chirag4601) [inline lib/Widgets/addTransporterWidget.dart:246]: when adding such print statements, use print like "error from updateShipperTransporterList method in lib/widgets/addTransporterWidget file: $e"

File: lib/Widgets/addTransporterWidget.dart [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+  }
+}
+
+Future<String> getTransporterIdByPhone(String phoneNo) async {
```
Comment (chirag4601) [inline lib/Widgets/addTransporterWidget.dart:250]: separate such apis calls from frontend code

File: lib/Widgets/loadDetailsWebWidgets/moveLoadConfirmationScreenButtonWidget.dart [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+class _moveLoadConfirmationScreenButtonWidgetState
+    extends State<moveLoadConfirmationScreenButtonWidget> {
+  @override
+  Widget build(BuildContext context) {
+    bool enable = false;
+    ProviderData providerData = Provider.of<ProviderData>(context);
+    print(providerData.truckTypeValue);
+    print(providerData.passingWeightMultipleValue);
```
Comment (chirag4601) [inline lib/Widgets/loadDetailsWebWidgets/moveLoadConfirmationScreenButtonWidget.dart:25]: remove these print statements

File: lib/screens/PostLoadScreens/loadConfirmation.dart [hunk 7]
```diff
@@ -28,153 +31,333 @@ class _LoadConfirmationState extends State<LoadConfirmation> {
...
+                                elevation: 5,
+                                child: Padding(
+                                  padding: EdgeInsets.fromLTRB(
+                                      space_3, space_2, space_3, space_3),
+                                  child: Column(
+                                    mainAxisAlignment: MainAxisAlignment.start,
+                                    children: [
+                                      Container(
+                                        child: LoadConfirmationTemplate(
+                                            value: providerData
```
Comment (chirag4601) [inline lib/screens/PostLoadScreens/loadConfirmation.dart:248]: this same code is used above also.
Can you try to store this value in a variable and use from there?

Comment (Anik-Chakraborty) [reply lib/screens/PostLoadScreens/loadConfirmation.dart:248]: There are two different designs. Above one for new dashboard and Second one is for previous mobile designs and some attributes are also different in both design.

**Pull request merged**